README.md

```diff
@@ -67,6 +67,27 @@ You should verify the .deb file with the associated .sig before you proceed any
 sudo dpkg -i dmd_2.084.0-0_amd64.deb
 ```
 
+#### Ubuntu (Bionic Stable)
+Use apt and install the following packages, if you haven't already:
+
+* libsdl2-2.0-0
+* libsdl2-image-2.0-0
+* libsdl2-mixer-2.0-0
```

Comment:
libsd2-mixer isn't needed when we have libcsfml-audio2.x, unless we want to compile for it; usually we only wish to do that if sfml isn't available for some reason.